eva-accession-import/pom.xml

```diff
@@ -40,6 +40,12 @@
             <artifactId>postgresql</artifactId>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>com.lordofthejars</groupId>
+            <artifactId>nosqlunit-mongodb</artifactId>
+            <version>1.0.0-rc.5</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
I have noticed this dependency is already declared in `eva-accession-core`. It should be moved to the parent POM, then declared (without version) in the children.